functions/index.js

```diff
@@ -7,18 +7,19 @@ const options = {
   verbose: !!process.env.FIREBASE_FUNCTIONS_EMULATOR,
   mappings: [
     {
-      from: /^\/toolbox/,
-      to: '/arcgis/rest/services/Electrofishing/Download/GPServer',
+      // the optional /maps prefix is to support the maps rewrite rule in firebase.json
```

Comment:
**question**: how is it optional?

Reply:
Because of the `?` immediately following the capture group.